src/components/App/App.js

```diff
@@ -1,75 +1,79 @@
-import React, { useState, useEffect } from 'react'
-import { Switch, Route } from 'react-router-dom'
-import Navbar from '../Navbar/Navbar'
-import Home from '../Home/Home'
-import './App.scss'
-import Dashboard from '../Dashboard/Dashboard'
-import UsersContext from '../../Context/UsersContext'
-import ErrorContext from '../../Context/ErrorContext'//Probably don't need an error context because App can conditionally render error if it's state exists
-import RequestRoomieContext from '../../Context/RequestRoomieContext'
-import CurrentUserContext from '../../Context/CurrentUserContext'
-import { getUsers } from '../../utilities/apiCalls'
+import React, { useState, useEffect } from "react"
+import { Switch, Route } from "react-router-dom"
+import Navbar from "../Navbar/Navbar"
+import Home from "../Home/Home"
 
-export default function App() {
+import dummyData from "../../utilities/dummyData"
 
-  const [users, setUsers] = useState([])
-  const [error, setError] = useState('')
-  const [currentUser, setCurrentUser] = useState(
-    {
-      "id": "13",
-      "type": "user",
-      "attributes": {
-        "roomies": [{id: '5', status: 'pending'}],
-        "email": "dusty@email.com",
-        "name": "The Dust",
-        "gender": "male",
-        "age": 90,
-        "location": {
-        "city": "Orlando",
-        "state": "FL"
-        }
-      }
-    }
-  )
+import "./App.scss"
+import Dashboard from "../Dashboard/Dashboard"
+import Login from "../Login/Login"
+import UsersContext from "../../Context/UsersContext"
+import ErrorContext from "../../Context/ErrorContext"
+import RequestRoomieContext from "../../Context/RequestRoomieContext"
+import CurrentUserContext from "../../Context/CurrentUserContext"
+import { getUsers } from "../../utilities/apiCalls"
 
+export default function App() {
+  const [users, setUsers] = useState([])
+  const [error, setError] = useState("")
+  const [currentUser, setCurrentUser] = useState(null)
 
   useEffect(() => {
     const fetchUsers = async () => {
       try {
-        const response = await getUsers();
-        setUsers(response.data);
+        const response = await getUsers()
+        setUsers(response.data)
       } catch (err) {
-        setError(err);
+        setError(err)
       }
-    };
-    fetchUsers();
-  }, []);
+    }
+    // fetchUsers()
```

Comment:
nice, this can get refactored once the endpoint is functional. but nice to use this in the meantime so we can move forward.